src/globus_sdk/_testing/data/compute/get_function.py

```diff
@@ -0,0 +1,24 @@
+from globus_sdk._testing.models import RegisteredResponse, ResponseSet
+
+from ._common import FUNCTION_CODE, FUNCTION_ID, FUNCTION_NAME
+
+FUNCTION_DOC = {
+    "function_uuid": FUNCTION_ID,
+    "function_name": FUNCTION_NAME,
+    "function_code": FUNCTION_CODE,
+    "description": "I just wanted to say hello.",
+    "metadata": {"python_version": "3.12.6", "sdk_version": "2.28.1"},
```

Comment:
Do these values need to stay up-to-date at all?

Or is it only important that there's "python_version" and" sdk_version" keys that point to strings?

Reply:
The latter: these indicate the Python and Compute SDK versions used to serialize the function, thus they should never change.